packages/aws-cdk-lib/aws-ses-actions/lib/workmail.ts

```diff
@@ -0,0 +1,45 @@
+import * as ses from '../../aws-ses';
+import * as sns from '../../aws-sns';
+
+/**
+ * Construction properties for a WorkMail action.
+ */
+export interface WorkMailProps {
+  /**
+   * The WorkMail organization ARN.
+   *
+   * @example 'arn:aws:workmail:us-east-1:123456789012:organization/m-organizationid'
```

Comment:
```suggestion
   *
   * Amazon WorkMail organization ARNs are in the form
   * `arn:aws:workmail:region:account_ID:organization/organization_ID`
   *
   * @example 'arn:aws:workmail:us-east-1:123456789012:organization/m-68755160c4cb4e29a2b2f8fb58f359d7'
```
Using the example from the documentation seems more convenient.